Client/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,201 @@
+﻿using AutoMapper;
+using Client.Models;
+using Client.Models.OrderEntities.Order;
+using Library.OrderEntities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Client.Controllers
+{
+    // Ideas:
+    // make use of [AllowAnonymous] to allows users fetch their orders
+    // make use of policies in the future 
+    // find a way to make a fetch request with current user id as a filter
+    [Authorize(Roles = "admin")] // Under question
+    public class OrdersController : Controller
+    {
+        private readonly HttpClient _httpClient;
+        private readonly string _baseAddress;
+        private readonly IMapper _mapper;
+        public OrdersController(HttpClient httpClient, IMapper mapper)
+        {
+            _httpClient = httpClient;
+            _baseAddress = "https://localhost:7007/gateway/orders";
```

Comment:
Port 7007 is listening for UserAPI, so lookup for OrderAPI/Properties/.json/https